www/lib/HTML/QuickForm/advmultiselect.php

```diff
@@ -390,117 +394,139 @@ function __construct($elementName = null, $elementLabel = null,
-            break;
-        case 'movedown':
-            if (is_null($attributes)) {
-                $this->_downButtonAttributes
+                } else {
+                    $this->_updateAttrArray(
+                        $this->_upButtonAttributes,
+                        $this->_parseAttributes($attributes)
+                    );
+                }
+                break;
+            case 'movedown':
+                if (is_null($attributes)) {
+                    $this->_downButtonAttributes
                     = array('name'  => 'down',
                             'value' => ' Down ',
                             'type'  => 'button');
-            } else {
-                $this->_updateAttrArray($this->_downButtonAttributes,
-                                        $this->_parseAttributes($attributes));
-            }
-            break;
-        case 'movetop':
-            if (is_null($attributes)) {
-                $this->_topButtonAttributes
+                } else {
+                    $this->_updateAttrArray(
+                        $this->_downButtonAttributes,
+                        $this->_parseAttributes($attributes)
+                    );
+                }
+                break;
+            case 'movetop':
+                if (is_null($attributes)) {
+                    $this->_topButtonAttributes
                     = array('name'  => 'top',
                             'value' => ' Top ',
                             'type'  => 'button');
-            } else {
-                $this->_updateAttrArray($this->_topButtonAttributes,
-                                        $this->_parseAttributes($attributes));
-            }
-            break;
-        case 'movebottom':
-            if (is_null($attributes)) {
-                $this->_bottomButtonAttributes
+                } else {
+                    $this->_updateAttrArray(
+                        $this->_topButtonAttributes,
+                        $this->_parseAttributes($attributes)
+                    );
+                }
+                break;
+            case 'movebottom':
+                if (is_null($attributes)) {
+                    $this->_bottomButtonAttributes
                     = array('name'  => 'bottom',
                             'value' => ' Bottom ',
                             'type'  => 'button');
-            } else {
-                $this->_updateAttrArray($this->_bottomButtonAttributes,
-                                        $this->_parseAttributes($attributes));
-            }
-            break;
-        default;
-            return PEAR::throwError('Argument 1 of HTML_QuickForm_advmultiselect::' .
+                } else {
+                    $this->_updateAttrArray(
+                        $this->_bottomButtonAttributes,
+                        $this->_parseAttributes($attributes)
+                    );
+                }
+                break;
+            default;
+            return PEAR::throwError(
```

Comment:
we don't use PEAR anymore (but it's in the quickform lib).

Reply:
This PR is for style fix only. I do not know how to replace this and IMHO this should be part of another PR.